tests/js/client/shell/shell-cluster-collection-selectivity.js

```diff
@@ -72,8 +72,7 @@ function ClusterCollectionSuite () {
       for (let i = 0; i < 10; ++i) {
         c.save({foo: i});
       }
-      // waitForEstimatorSync does nothing in the cluster but waiting...
-      internal.waitForEstimatorSync(); // make sure estimates are consistent
+      arango.POST("/_admin/execute", "require('internal').waitForEstimatorSync();");  // make sure estimates are consistent
```

Comment:
I think my PR to fix your first series already has a better way of doing that via an API instead of remote js execution. 
Please refer to my PR.